src/renderers/dom/fiber/ReactDOMFiberComponent.js

```diff
@@ -200,81 +200,68 @@ function trapBubbledEventsLocal(inst) {
     'trapBubbledEvent(...): Requires node to be rendered.'
   );
 
+  // TODO: Make sure that we check isMounted before firing any of these events.
```

Comment:
Is this on a task list somewhere? If it isn't working properly I know we'll get weird bugs in www.